app/models/monologue/post.rb

```diff
@@ -44,6 +44,14 @@ def self.page p
     self.limit(per_page).offset(offset)
   end
 
+  def self.listing_page(p)
+    per_page = Monologue::Config.admin_posts_per_page || 50
+    set_total_pages(per_page)
+    p = (p.nil? ? 0 : p.to_i - 1)
+    offset =  p * per_page
```

Comment:
could you create a private method and reuse it. That code is exactly the same as the one in the method self.page.
Thanks